src/Microsoft.AspNetCore.Razor/Parser/TagHelpers/TagHelperBlockBuilder.cs

```diff
@@ -29,8 +29,7 @@ public TagHelperBlockBuilder(TagHelperBlock original)
 
         /// <summary>
         /// Instantiates a new instance of the <see cref="TagHelperBlockBuilder"/> class
-        /// with the provided <paramref name="tagName"/> and derives its <see cref="Attributes"/>
-        /// and <see cref="BlockBuilder.Type"/> from the <paramref name="startTag"/>.
+        /// with the provided values.
```

Comment:
Because the comment was inaccurate. There is no `startTag` param anymore and we no longer derive Attributes from or BlockBuilder.Type from it.